sys/log/full/src/log.c

```diff
@@ -492,6 +494,18 @@ log_append_prepare(struct log *log, uint8_t module, uint8_t level,
 #else
     assert(etype == LOG_ETYPE_STRING);
 #endif
+#if MYNEWT_VAL(LOG_VERSION) > 3
+
+#if MYNEWT_VAL(LOG_FLAGS_LOG_IMG_HASH)
+    uint8_t hash[IMGMGR_HASH_LEN];
+
+    ue->ue_flags = LOG_FLAGS_LOG_IMG_HASH_e;
+
+    imgr_read_info(0, NULL, hash, NULL);
+    memcpy(ue->ue_rsvd, hash, 4);
+#endif // MYNEWT_VAL(LOG_FLAGS_LOG_IMG_HASH)
```

Comment:
Our coding_style says we do not allow C++ comments. Please remove those or change them into C style comments using `/* */`.

Reply:
Removed.